packages/did/src/Did.chain.ts

```diff
@@ -88,6 +99,11 @@ export async function queryEndpointsCountsEncoded(
   return api.query.did.didEndpointsCount<u32>(didIdentifier)
 }
 
+async function queryDepositAmountEncoded(): Promise<U128> {
```

Comment:
Same as above. If it is needed, then this should be renamed to follow the convention of being `raw` amount.

Reply:
You are right. I will rename everything as encoded.